assets/admin/postcss.config.js

```diff
@@ -0,0 +1,2 @@
+/* reuse the postcss configuration included in sulu */
+module.exports = require('../../vendor/sulu/sulu/postcss.config.js');
```

Comment:
@danrot WDYT? Should we reuse the Sulu config per default or should we just copy the content? 

To me, both options would be okay. But we need to think about updating the config in the `skeleton` if we choose to copy it. Otherwise it will be confusing when developers try to copy code from the Sulu code.

Reply:
I am a bit torn on this... And there is also currently a bit of a mix in the skeleton, although I can understand why that is.

The **`webpack.config.js` imports the config** from the sulu/sulu repository, which makes sense, because you can only have a single webpack config for the entire project, and it needs to fit the sulu/sulu one, because otherwise the build does not work.

The **`.babelrc` file copied the config** for probably two reasons:
1. The format we have chosen (I think it was also the only one when we started) does not support any imports
2. Babel also allows to have a different babel config per folder. So with the structure as is enables the developer to have a different babel config from the one in sulu/sulu.

Additionally copying makes it less likely that a project breaks, because of changes we introduce in the sulu/sulu config. Especially for this reason I would probably also go with copying again. It allows to have a completely different postcss config, if the project team decides to use more postcss plugins or does not want to use some of the ones we've configured. What we've got in sulu/sulu might be a sensible default, so I would paste that one in here.

Mind that I am assuming that the `postcss.config.js` file is also only valid for the folder it is located in, maybe that should be doublechecked before making that decision.

Reply:
Alright - copying the file is okay for me 🙂

Do we need to add the `postcss` packages to the `assets/admin/package.json` too when adding such a config?
I guess it will work anyway, but what would be the correct thing to do?

Reply:
They should be listed in order to keep working if we decide to remove some of these packages at some point. But I think that should already be the case, isn't it?

Reply:
You are right. I must have overlooked that before 🙈  I adjusted the PR to copy the content of the file from Sulu